app/src/test/kotlin/compiler/intermediate/FunctionControlFlowTest.kt

```diff
@@ -601,4 +674,254 @@ class FunctionControlFlowTest {
 
         assertEquals(listOf<Diagnostic>(ControlFlowDiagnostic.Warnings.UnreachableStatement(evaluation)), diagnostics)
     }
+
+    @Test
```

Comment:
For readability and to keep consistency with other test, we can add in comments how does the code look like, eg.
```
przekaźnik generator() {}
f() {
    otrzymując x: Liczba od generator() {123}
}
```